frontend/src/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
The app is not compiling for me when I run off this branch, since prettier complains of CRLF in the following files. Perhaps you have some unsaved changes to push?

- frontend\src\index.tsx
- frontend\src\App.tsx
- frontend\src\components\Dropzone.tsx
- frontend\src\components\Navbar.tsx
- frontend\src\components\SelectedMenu.tsx
- frontend\src\reportWebVitals.ts
- frontend\src\theme.ts

Reply:
Huh, I'm not sure how to address this issue. After I ran our PowerShell script, all CRLF end of line changed to LF in my VS Code, but Git didn't track these changes at all, so there was no opportunity to make a commit. I also double checked each of the files you listed in this comment, and they're all LF for me. I wonder if we each have to make the adjustment locally? In the meantime, I'm going to search for a better solution.

Reply:
Helpful links: 
- https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings
- https://stackoverflow.com/questions/2825428/why-should-i-use-core-autocrlf-true-in-git